extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -68,6 +70,29 @@ amp-story:not([supports-landscape]) + amp-subscriptions-dialog.i-amphtml-story-s
   background: #EDEDED !important;
 }
 
+.i-amphtml-story-subscriptions-dialog-banner-button {
+  height: 100% !important;
+  display: flex !important;
+  justify-content: flex-end !important;
+  align-items: center !important;
+  letter-spacing: 0.015em !important;
+  margin-right: 0.58em !important;
```

Comment:
This does not support RTL.
Please move this to the parent and use padding. It needs to be on left and right of the container.
That way no matter which side the content is on it will have the same padding.